model/config.go

```diff
@@ -43,6 +43,9 @@ type ConfigMeta struct {
 	// (security domains, fault domains, organizational domains)
 	Namespace string `json:"namespace,omitempty"`
 
+	// Namespace where istio control plane is installed
+	IstioNamespace string `json:"istioNamespace,omitempty"`
```

Comment:
Is this really necessary as a general config metadata? This struct is mostly a copy of k8s ObjectMeta and this is deviating from that.